README.md

```diff
@@ -343,6 +344,65 @@ More information about the `@include` directive and its options can be found her
 
 For information on working with locales, please refer to the [Translations README](./scripts/translations/README.md).
 
+### Expanding Wildcard Domains
+
+You can expand wildcard domains using the CLI commands provided in this project. These commands help update wildcard domains and expand wildcard domains for various platforms.
+
+1. **Install Dependencies**
+
+    ```bash
+    yarn install
+    ```
+
+1. **Update Wildcard Domains**
+
+   This command extracts wildcard domains from the filters, converts them to a list of domains, selects the alive domains, and saves the resulting map to a JSON file [wildcard_domains.json](scripts%2Fwildcard-domain-processor%2Fwildcard_domains.json).
+
+    ```bash
+    yarn update-wildcard-domains <filtersDir> <wildcardDomainsFile>
```

Comment:
it is not clear when this command should be run? if not on cron and only manually, how often?

Reply:
it will be decided later